translations/binaries/slps_strings_items_consumable.json

```
@@ -0,0 +1,2153 @@
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1767496,
+            "string": "[Sign_Apple_Gel]アワーグラス[END]",
+            "translation": "[Sign_Apple_Gel]모래시계[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767432,
+            "string": "一定時間全ての敵の動きを止める。[LINE]\nピンチのときに使うのもアリ！[LINE]\n[END]",
+            "translation": "일정 시간 모든 적의 움직임을 멈춘다.[LINE]\n위기일 때 사용하는 것도 좋다![LINE]\n[END]",
+            "original_bytes_length": 63,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1767408,
+            "string": "[Sign_9]オベロナミンＣ[END]",
+            "translation": "[Sign_9]오베로나민C[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767320,
+            "string": "オベロン社が開発した庶民の味方。[LINE]\n最大ＨＰの４０％分ＨＰが回復する。[LINE]\n元気ハツラツ！[END]",
+            "translation": "오베론사가 개발한 백성의 친구.[LINE]\n최대 HP의 40%만큼 HP가 회복된다.[LINE]\n건강 활력 충전![END]",
+            "original_bytes_length": 83,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1767296,
+            "string": "[Sign_9]オベロナミンＥＸ[END]",
+            "translation": "[Sign_9]오베로나민EX[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767200,
+            "string": "オベロン社が開発した働く人の味方。[LINE]\n最大ＨＰの６０％分ＨＰが回復する。[LINE]\n疲労時の栄養補給に。[END]",
+            "translation": "오베론사가 개발한 직장인의 친구.[LINE]\n최대 HP의 60%만큼 HP가 회복된다.[LINE]\n피로 시 영양 보충에 좋다.[END]",
+            "original_bytes_length": 91,
+            "available_bytes_size": 95
+        },
+        {
+            "offset": 1767168,
+            "string": "[Sign_9]オベロナミンＧＯＬＤ[END]",
+            "translation": "[Sign_9]오베로나민GOLD[END]",
+            "original_bytes_length": 26,
+            "available_bytes_size": 31
+        },
+        {
+            "offset": 1767080,
+            "string": "オベロン社が開発した戦う人の味方。[LINE]\nＨＰが全快するドリンク。[LINE]\n今日のあなたに、この１本！[END]",
+            "translation": "오베론사가 개발한 전투원의 친구.[LINE]\nHP가 전체 회복되는 음료.[LINE]\n오늘의 당신에게, 이 한 병![END]",
+            "original_bytes_length": 87,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1767056,
+            "string": "[Sign_9]アイスキャンディー[END]",
+            "translation": "[Sign_9]아이스캔디[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24
+        },
+        {
+            "offset": 1766992,
+            "string": "冷たくてあま～い氷菓子。[LINE]\n最大ＨＰの１０％分回復する氷菓子。[LINE]\n[END]",
+            "translation": "차갑고 달콤한 얼음과자.[LINE]\n최대 HP의 10%만큼 회복하는 얼음과자.[LINE]\n[END]",
+            "original_bytes_length": 61,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1766968,
+            "string": "[Sign_9]ホーリィボトル[END]",
+            "translation": "[Sign_9]성스러운 병[END]",
```

```suggestion
            "translation": "[Sign_9]성령의 병[END]",
```

ToD2에서 성령의 병으로 번역되었다고 함